### PR TITLE
NAS-101886 / 11.3 / Prevent cutting long lines pasted into terminal

### PR DESCRIPTION
**Pull request by themylogin**

### CODE REVIEW

File: src/middlewared/middlewared/main.py [hunk 1]
```diff
@@ -550,12 +551,11 @@ def run(self):
         self.shell_pid, master_fd = os.forkpty()
         if self.shell_pid == 0:
             for i in range(3, 1024):
-                if i == master_fd:
-                    continue
```
Comment (william-gr) [inline src/middlewared/middlewared/main.py]: Why this change?

Comment (themylogin) [reply src/middlewared/middlewared/main.py]: It was useless, in child process `master_fd` is `-1`

Comment (william-gr) [reply src/middlewared/middlewared/main.py]: In that case we should use bsd.closefrom, which is much better than iterating 1021 times

**Comment by william-gr** [comment]
@themylogin are you comfortable with having this merged into 11.2? what are the chances something with closefrom breaks or the terminal that does work quite right?

I would be fine with only 11.3 fix

**Comment by themylogin** [comment]
I am, but since no one really complained about it, let's not tempt our fate.
